test/io/jesi/backpack/macros_test.cljc

```diff
@@ -272,3 +274,100 @@
                 expected (apply + args)
                 actual (apply impl args)]
             (is= expected actual)))))))
+
+(deftest with-open-test
+
+  (testing "closes the resource"
+    (let [ran (volatile! false)
+          o (->Closey)]
+      (with-open [o o]
+        (when (is (false? (closed? o)))
+          (vreset! ran true)))
+      (is (true? (closed? o))))
+
+    (testing "on exception"
+      (let [o (->Closey)]
+        (is (thrown? ExceptionInfo (with-open [o o]
+                                     (when (is (false? (closed? o)))
+                                       (throw (ex-info "Don't panic" {}))))))
+        (is (true? (closed? o)))))))
+
+;FIXME complete
+(deftest with-open->-test)
+
+
+(defn- identical?
+  ([x y] (clojure.core/identical? x y))
+  ([x y & more]
+   (->> (list* x y more)
+        (partition 2 1)
+        (every? (fn [[x y]]                                 ;TODO point-free
+                  (clojure.core/identical? x y))))))
+
+(deftest assoc-nx-test
+  (let [m1 {:a 1}
+        m2 (assoc m1
+             :b 2)
+        m3 (assoc m2
+             :c 3)]
+
+    #?(:clj (testing "is a macro"
+              (is (macro? `assoc-nx))))
+
+    (testing "expands"
+      ;Fails even though the content is the same (yeah that again)
```

Comment:
I'm getting sick of this issue. Will need ✋🧠👀

Reply:
hmmm.